src/utils/resizeImage.js

```diff
@@ -0,0 +1,28 @@
+import { getImageScaleRatio } from 'src/utils/getImageScaleRatio'
+
+/**
+ * @param {string} fileDataUri
+ * @param {string} fileType
+ * @returns {Promise<string>}
+ */
+export const resizeImage = async (fileDataUri, fileType) => {
+  return new Promise((resolve, reject) => {
+    const newImage = new Image()
+    newImage.src = fileDataUri
+    newImage.onerror = reject
+    newImage.onload = () => {
+      const canvas = document.createElement('canvas')
+      const scaleRatio = getImageScaleRatio(newImage)
+      const scaledWidth = scaleRatio * newImage.width
+      const scaledHeight = scaleRatio * newImage.height
+
+      canvas.width = scaledWidth
+      canvas.height = scaledHeight
+      canvas
+        .getContext('2d')
+        .drawImage(newImage, 0, 0, scaledWidth, scaledHeight)
+
+      resolve(canvas.toDataURL(fileType))
+    }
+  })
+}
```

Comment:
nit: It's weird to have one method per file. I would have a created a utils/pdf.js with all the PDF related methods, and why not on utils/image.js to store image's related methods for instance